k8s/examples/vineyard-csidriver/README.md

```diff
@@ -0,0 +1,205 @@
+$ kubectl delete csidrivers.k8s.v6d.io csidriver-sample
+```
+
+Delete the vineyard cluster:
+
+```bash
+$ vineyardctl delete vineyard-cluster
+```
+
+Delete the data volume:
+
+```bash
+$ kubectl delete -f prepare-data.yaml
+```
+
+## Result Analysis
+
+The time of argo workflow execution of the pipeline is as follows:
+
+### Argo workflow duration
+
+|    data scale   | without vineyard | with vineyard |
+| --------------- | ---------------- | ------------- |
+|     8500 Mi     |    186s          |      169s     |
+|     12000 Mi    |    250s          |      203s     |
+|     15000 Mi    |    332s          |      286s     |
+
+
+Actually, the cost time of argo workflow is affected by lots of factors, such as the network, the cpu and memory of the cluster, the data volume, etc. So the time of argo workflow execution of the pipeline is not stable. But we can still find that the time of argo workflow execution of the pipeline with vineyard is shorter than that without vineyard.
+
+Also, we record the whole execution time via logs. The result is as follows:
+
+### Actual execution time
+
+|    data scale   | without vineyard | with vineyard |
+| --------------- | ---------------- | ------------- |
+|     8500 Mi     |    139.3s        |      92.3s    |
+|     12000 Mi    |    204.3s        |      131.1s   |
+|     15000 Mi    |    289.3s        |      209.7s   |
+
+According to the above results, we can find that the time of actual execution of the pipeline with vineyard is shorter than that without vineyard. To be specific, we record the write/read time of the following steps:
+
+### Write time
+
+|    data scale   | without vineyard | with vineyard |
+| --------------- | ---------------- | ------------- |
+|     8500 Mi     |    21s           |      5.4s     |
+|     12000 Mi    |    26s           |      7s       |
+|     15000 Mi    |    32.2s         |      9.4s     |
+
+From the above results, we can find that the time of write of the pipeline with vineyard is nearly 4 times shorter than that without vineyard. The reason is that the data is stored in the vineyard cluster, so it's actually a memory copy operation, which is faster than the write operation of the nfs volume.
+
+
+### Read time(Delete the time of init data loading)
+
+|    data scale   | without vineyard | with vineyard |
+| --------------- | ---------------- | ------------- |
+|     8500 Mi     |    36.7s         |      0.02s    |
+|     12000 Mi    |    45.7s         |      0.02s    |
+|     15000 Mi    |    128.6s        |      0.04s    |
+
+Based on the above results, we can find that the read time of vineyard is nearly a constant, which is not affected by the data scale. The reason is that the data is stored in the shared memory of vineyard cluster, so it's actually a pointer copy operation.
+
+As a result, we can find that with vineyard, the argo workflow duration of the pipeline is reduced by 10%~20% and the actual execution time of the pipeline is reduced by about 30%.
```

Comment:
Put the readme to the tutorial directory , same as what you have done for another "e xample" under the `k8s/` directory:

https://github.com/v6d-io/v6d/tree/main/docs/tutorials/kubernetes